pkg/install/v1alpha1/jiva_volume.go

```diff
@@ -657,13 +659,19 @@ spec:
     action: list
     options: |-
       labelSelector: openebs.io/replica=jiva-replica,openebs.io/persistent-volume={{ .Volume.owner }}
-    retry: "12,10s"
+    retry: "24,5s"
```

Comment:
will there be issue for this long duration as service is already created?

Reply:
no